gammapy/spectrum/tests/test_models.py

```diff
@@ -250,10 +258,22 @@ def test_models(spectrum):
     if "e_peak" in spectrum:
         assert_quantity_allclose(model.e_peak, spectrum["e_peak"], rtol=1e-2)
 
-    # inverse for ConstantModel is irrelevant
-    if not (isinstance(model, ConstantModel) or spectrum["name"] == "compound6"):
+    # inverse for ConstantModel is irrelevant.
+    # inverse for Gaussian has a degeneracy
+    if not (
+            isinstance(model, ConstantModel)
+            or spectrum["name"] == "compound6"
+            or spectrum["name"] == "Gaussian"
+    ):
         assert_quantity_allclose(model.inverse(value), 2 * u.TeV, rtol=0.01)
 
+    if "integral_infinity" in spectrum:
+        emin = -10000 * u.TeV
```

Comment:
Shouldn't it be 0?

Reply:
for me the test is to check if it is equal to the norm when you integrate from -inf to +inf, no?

Reply:
maybe you are right and I can check that from 0 to inf is equal to No/2 as you prefer

Reply:
I just meant that you don't want a negative energy. So you take a small width gaussian and integration from 0 to +inf should be the same